en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>name</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>owner</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application owner name.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### CacheHit
+
+Represents response cache hits and misses values for a single timestamp. Timestamp is calculated according to the
+provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
+granularity value is '1d', timeSpan will represents start of the day calculated adhering to the timezone provided in
+the TimeFilter.
+Ex: timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>hits</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Number of times response cache hit .
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>misses</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Number of times response cache missed.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>hitPercentage</strong></td>
+<td valign="top"><a href="#float">Float</a>!</td>
+<td>
+
+Response cache hits as a percentage to total of hits and misses.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### CacheHits
+
+Represents response cache hits and misses over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#cachehit">CacheHit</a>]</td>
+<td>
+
+List containing cache hits, misses and hit percentage over time. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### DetailsOfErrors
+
+Represents error details over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#errordetails">ErrorDetails</a>]!</td>
+<td>
+
+List containing error details over time. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".